content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -155,145 +124,146 @@ enum Level {
 
-      If you see, "The database needs to be empty to proceed", you can simply use a different database name. Using the screenshot above, that would be renaming `Log` to something else.
+1. Add the Prisma Accelerate connection string to your local environment `.env` file and the `DIRECT_URL` environment variable to perform migrations and introspections.
 
-      </Admonition>
-
-      If all goes well, you'll be greeted with a new connection string that starts with `prisma://`.
-
-   5. Copy the `prisma://` connection string to your clipboard.
-
-      ![Data Proxy Page](./images/450-05-data-proxy.png)
-
-## 7. Set the Data Proxy Connection string in your environment
+   ```diff file=.env
+   -DATABASE_URL="postgres://..."
+   +DATABASE_URL="prisma://accelerate.prisma-data.net/?api_key=__API_KEY__"
+   +DIRECT_URL="postgres://..."
+   ```
 
-1. Add the Data Proxy connection string to your local environment `.env` file.
+   Add [`directUrl`](/reference/api-reference/prisma-schema-reference#fields) property in `datasource` block in the `schema.prisma` file.
 
-   ```diff file=.env
-   - DATABASE_URL = "mongodb+srv://<user>:<password>@cluster.mongodb.net/Log?retryWrites=true&w=majority"
-   + DATABASE_URL = "prisma://aws-us-east-1.prisma-data.com/?api_key=•••••••••••••••••"
+   ```diff file=prisma/schema.prisma
+   datasource db {
+    provider = "postgresql"
+    url = env("DATABASE_URL")
+    +directUrl = env("DIRECT_URL")
+   }
    ```
 
+> The `directUrl` field is not required for Prisma Accelerate. It allows you to easily perform database migrations and introspections.
+
 2. In your code editor, add the connection string in the `wrangler.toml` file.
 
    ```diff file=wrangler.toml
-     name = "prisma-mongodb-cloudflare"
+     name = "prisma-cloudflare-accelerate"
      main = "src/main.ts"
      compatibility_date = "2022-11-07"
 
    + [vars]
-   + DATABASE_URL = "prisma://aws-us-east-1.prisma-data.com/?api_key=•••••••••••••••••"
+   + DATABASE_URL = "prisma://accelerate.prisma-data.net/?api_key=__API_KEY__"
+   ```
+
+3. Install the Accelerate Client extension
+
+   ```bash
+   npm install @prisma/extension-accelerate
    ```
 
 You are now ready to generate a Prisma Client.
 
-## 8. Generate a Prisma Client
+## 6. Migrate your schema and generate a Prisma Client
 
-Next, you'll generate a Prisma Client that connects through the [Data Proxy](/data-platform/classic-projects/data-proxy) over HTTP.
+To apply migrations execute the following command:
+
+```
+prisma migrate deploy
+```
+
+Next, you'll generate Prisma Client that connects to your database through [Prisma Accelerate](/data-platform/accelerate) over HTTP.
 
 ```terminal
 npx prisma generate --no-engine
 ```
 
 <Admonition>
 
-Prior to Prisma 5.2.0, the `--no-engine` flag is not available. Instead, use the `--data-proxy` flag.
+Prior to Prisma 5.2.0, the `--no-engine` flag is not available. Instead, use the `--accelerate` flag.
```

Comment:
If we expect a user to initialize the project from scratch, the latest Prisma version should be installed

```suggestion
The `--no-engine` flag is available from Prisma 5.2.0 and later. If you're using an earlier version of Prisma, use the `--accelerate` flag.
```